tensornetwork/ncon_interface_test.py

```diff
@@ -18,121 +18,148 @@
 import pytest
 import numpy as np
 import tensorflow as tf
-tf.compat.v1.enable_v2_behavior()
 from tensornetwork import ncon_interface
 from tensornetwork.contractors.naive_contractor import naive
+from tensornetwork.backends import backend_factory
+from jax.config import config
+config.update("jax_enable_x64", True)
+tf.compat.v1.enable_v2_behavior()
+
+
+#TODO: fix jax backend and add to fixture
```

Comment:
We should fix that now. What was the issue?

Reply:
Remove this comment as it's mostly fixed now.

Reply:
removed